src/app/components/Input/InputCurrency.tsx

```diff
@@ -1,68 +1,44 @@
 import { Currency } from "@arkecosystem/platform-sdk-intl";
-import React, { useCallback, useEffect, useLayoutEffect, useRef, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 
 import { Input } from "./Input";
 
 type Props = {
 	addons?: any;
 	onChange?: (value: any) => void;
-	magnitude?: number;
 	as?: React.ElementType<any>;
 } & Omit<React.InputHTMLAttributes<any>, "onChange" | "defaultValue">;
 
-interface ISelectionRange {
-	start: number | null;
-	end: number | null;
-}
+const sanitizeValue = (x?: string) => Currency.fromString(x || "").display;
```

Comment:
moved this outside the component as it's a pure function, not sure its the best approach tho